solc/CommandLineParser.cpp

```diff
@@ -77,6 +77,7 @@ static string const g_strInputFile = "input-file";
 static string const g_strInterface = "interface";
 static string const g_strYul = "yul";
 static string const g_strYulDialect = "yul-dialect";
+static string const g_strDebugInfo = "debug-info";
```

Comment:
In the end I called it `--debug-info` rather than `--ir-debug-info` because it affects EVM assembly as well.